doc/man/8/ceph.rst

```diff
@@ -506,11 +506,11 @@ Usage::
 
 	ceph osd blocked-by
 
-Subcommand ``create`` creates new osd (with optional UUID).
+Subcommand ``create`` creates new osd (with optional UUID and ID).
```

Comment:
Added. But to add-or-rm-osds.rst, not in the man page, because the man looks much more laconic and I don't want to change its style.

Also I am not sure about my English, so if you see how the text can be improved, please tell me.